src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -1696,27 +1933,25 @@ public async Task TestIndexerCall1()
         [Fact, Trait(Traits.Feature, Traits.Features.TypeInferenceService)]
         public async Task TestIndexerCall2()
         {
-            // Update this when binding of indexers is working.
```

Comment:
Since I think binding of indexers are now working, I updated the test to be more appropriate I think.